<?xml version="1.0"?>
<launch>

  <node name="random_move" pkg="gen_dataset" type="random_move_euler.py"/>
  
  <arg name="num_dataset" default="1000" />
 
  <arg name="sensor_parent_frame" default="photoneo_center_optical_frame" />
  <arg name="topic_name" default="" />
  <arg name="object_name" default="HV8" />
  <arg name="object_up" default="HV8_up" />
  <arg name="init_pose_up" default="-x 1 -y 0 -z 0.5 -R 0.0 -Y -0.0" />
  <arg name="init_x" default="1" />

  <node pkg="gen_dataset" type="record_data.py" name="record_data" output="screen" >
    <param name="sensor_parent_frame" value="$(arg sensor_parent_frame)" />
    <param name="topic_name" value="$(arg topic_name)" />
    <param name="object_name" value="$(arg object_name)" />
    <param name="num_dataset" value="$(arg num_dataset)" />
  </node>

  <node name="zero_gravity" pkg="tf_publish" type="zero_gravity.sh" output="screen"/>
  
 
</launch>
